### DISCUSSION TITLE
Can I use an SSL certificate from a trusted CA on mitmproxy?

### DISCUSSION THREAD

**Discussion by fairchilddb** [mitmproxy/mitmproxy#7126] [Q&A]
Copy question from Stack Overflow
https://stackoverflow.com/questions/64256031/can-i-use-an-ssl-certificate-from-a-trusted-ca-on-mitmproxy
When use a domain name and SSL cert from a trusted CA,is it possible to interprete SSL traffic without getting warning like using self signed SSL cert?

**Comment by mhils** [comment]
Duplicate of https://github.com/mitmproxy/mitmproxy/discussions/7113
